dataland-e2etests/src/test/kotlin/org/dataland/e2etests/tests/communityManager/SingleDataRequestsTest.kt

```diff
@@ -246,7 +248,7 @@ class SingleDataRequestsTest {
             frameworkName = SingleDataRequest.FrameworkName.lksg,
             listOfReportingPeriods = listOf("2022"),
         )
-        jwtHelper.authenticateApiCallsWithJwtForTechnicalUser(TechnicalUser.Uploader)
+        jwtHelper.authenticateApiCallsWithJwtForTechnicalUser(TechnicalUser.PremiumUser)
```

Comment:
This line is in principle not needed anymore due to the before-all hook where the premium user is authenticated

Reply:
correct - deleted it